hiplot/plot_study.ipynb

```diff
@@ -0,0 +1,189 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://github.com/optuna/optuna-examples/blob/main/hiplot/plot_study.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Visualizing High-dimensional Parameter Relationships\n",
+    "\n",
+    "This notebook demonstrates the visualization of `study` in Optuna with [HiPlot](https://facebookresearch.github.io/hiplot/).\n",
+    "The hyperparameters of a neural network trained to classify images are optimized and the resulting study is then visualized using these features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If you run this notebook on Google Colaboratory, uncomment the below to install Optuna and HiPlot.\n",
+    "#! pip install --quiet optuna hiplot"
```

Comment:
```suggestion
    "#! pip install --quiet optuna hiplot scikit-learn pandas"
```

How about adding scikit-learn to the notebook as well for making it self-contained (for colab)?

Reply:
maybe like this?

```jupyter
    "# If you run this notebook on Google Colaboratory, uncomment the below to install Optuna and HiPlot.\n",
    "#! pip install --quiet optuna hiplot"
    "# If you run this notebook on your local, uncomment the below to install Optuna, HiPlot, and the other depedencies.\n",
    "#! pip install --quiet optuna hiplot scikit-learn pandas"
```

Reply:
Yes, it is just what I expected.

Reply:
`Google Colaboratory` -> `your local`

Do you have any intention to remove colab mention?

Reply:
The comment above is not a diff version: all four lines are written in the cell.

Reply:
Ah, it makes sense.